srcpkgs/tageditor/template

```diff
@@ -0,0 +1,63 @@
+# Template file for 'tageditor'
+pkgname=tageditor
+version=3.3.4
+revision=1
+build_style=cmake
+configure_args="-DBUILD_SHARED_LIBS:BOOL=ON"
+makedepends="tagparser-devel c++utilities-devel"
+short_desc="Tag editor for AAC, ID3, Vorbis, Opus, FLAC and Matroska (CLI only)"
+maintainer="Andrew J. Hesford <ajh@sideband.org>"
+license="GPL-2.0-only"
+homepage="https://github.com/Martchus/${pkgname}"
+distfiles="https://github.com/Martchus/${pkgname}/archive/v${version}.tar.gz"
+checksum=0874301bd7644a312d259d54a8b60366c0db80e5b1179c1039cad9905c95463d
+build_options="qt5 webengine webkit script"
+build_options_default="qt5"
+
+desc_option_webengine="Enable support for qt5-webengine file views (requires qt5)"
+desc_option_script="Enable support for qt5-script javascript (requires qt5)"
+desc_option_webkit="Enable support for qt5-webkit file views (requires qt5,~webengine)"
```

Comment:
vopt_conflict

Reply:
Thanks for the tip, vopt_conflict is now used.